examples/resources/awscc_sagemaker_image/sagemaker_image.tf

```diff
@@ -0,0 +1,4 @@
+resource "awscc_sagemaker_image" "example" {
+  image_name     = "example"
+  image_role_arn = aws_iam_role.test.arn
```

Comment:
Thank you for contribution, @vivekthangamuthu . 
- Could you update the `image_role_arn` to be a variable or a reference of an AWSCC resource?
- And, if you can please include the tags as in other examples.

Reply:
@quixoticmonk Thanks i have fixed the same now.